### ISSUE TITLE
MUF FORCE for mlev < 4 as SCARY_MUF_PRIM (SF issue)

### ISSUE THREAD

**Issue by tanabi** [fuzzball-muck/fuzzball#633]
I'd like to see FORCE opened up for muckers with less
than wizbits, with the following restrictions:

M1 and M2 would only be able to FORCE players and
things that the program's owner controls().
The forcelock check for these levels would be: program
dbref, current program uid.
(for M1, that means that the program dbref or the
program's owner's dbref would need to be in the
forcelock, as well as the object to be forced being
controlled by the program's owner.)
(for M2, that means that the program dbref or the
program uid's dbref [as affected by SETUID and HARDUID]
would need to be in the forcelock, as well as the
object to be forced being controlled by the program's
owner.)

M3 would be able to FORCE any player or things owned by
anyone, subject to the forcelock check against:
program dbref
current program uid (as affected by SETUID and HARDUID).

M4 would continue to be able to FORCE anything.

Opening FORCE in this fashion would be a
SCARY_MUF_PRIMS primitive.

**Comment by aidanPB** [comment]
Since the removal of ``SCARY_MUF_PRIMS``, this would presumably be controlled/enabled by a GOD-only tuneparam.

**Comment by wyld-sw** [comment]
Yeah, the name of the issue was copied from source forge, where it originated.
